src/appleseed-max-impl/logtarget.cpp

```diff
@@ -46,6 +46,34 @@
 
 namespace asf = foundation;
 
+#define WM_TRIGGER_CALLBACK WM_USER+4764
+
+namespace
+{
+  struct message_data
+  {
+      std::vector<std::string> lines;
+      DWORD type;
+  };
+
+  void ui_log_writer(UINT_PTR param_ptr)
+  {
+     message_data* data_ptr = reinterpret_cast<message_data*>(param_ptr);
+
+    for (auto line : data_ptr->lines)
+    {
+      GetCOREInterface()->Log()->LogEntry(
```

Comment:
Where does this write messages, actually? It sounds interesting.

Reply:
It writes to Max.log file in %localappdata%\Autodesk\3dsMax\2016 - 64bit\ENU\Network
Is this what you are asking?